tests/views_test.py

```diff
@@ -1,5 +1,100 @@
-from src_api.views import helloworld
+from numpy import ndarray, result_type
+from src_api.views import *
+import shutil
+import os
 
+def TestBasics():
+    """
+    Sets the Basic Folder/Files for Testing.
 
-def test_helloworld():
-    assert helloworld() == "Hello world !"
+    Returns:
+        tuple: filenames and folder.
+    """
+    folder_name = os.path.join(app.config["APP_FOLDER"], "tests", "testfiles")
+    file_name = "test_file.wav"
+    file_name_copy = "test_file_copy.wav"
+    return folder_name,file_name,file_name_copy
+
+
+def TestBasicsCopy(shutil):
+    """
+    Creates Copies of the Files from TestBasics() for Testing.
+    
+    Returns:
+        tuple:  filenames and folder.
+    
+    """
+    folder_name, file_name, file_name_copy = TestBasics()
+    shutil.copyfile(os.path.join(folder_name,file_name), os.path.join(folder_name,file_name_copy))
+    return folder_name,file_name_copy
+
+
+def test_predict_from_file():
+    print()
+
+
+def test_backend_save_request_object():
+    """
+    filetype?
+    """
+    print()
+
+
+def test_backend_rawfile_load():
+    print()
+    
+
+def test_backend_file_load():
+    print()
+
+
+def test_get_duration():
+    print()
+
+
+def test_get_sampling_rate():
```

Comment:
i would suggest a few more test on this function using different files with different sampling rates to ensure the function returns the correct sampling rate each time.
I would suggest the use of pytest parametrize for this